### PR TITLE
Fixed kode/#44: replace KDSOAP_DEBUG_PARSER

### PR DESCRIPTION
**Pull request by martonmiklos**
Replaced conditioned qDebugs with qCDebug

### CODE REVIEW

File: schema/parser.cpp [hunk 1]
```diff
@@ -648,7 +640,7 @@ Attribute Parser::parseAttribute(ParserContext *context, const QDomElement &elem
             newAttribute.setAttributeUse(Attribute::Required);
         } else if (use == QLatin1String("prohibited")) {
             qWarning("prohibited attributes are not supported"); // TODO skip parsing the attribute
-                                                                 // altogether
+            // altogether
```
Comment (dfaure-kdab) [inline schema/parser.cpp:643]: the previous indentation made more sense (maybe merge the lines?)

File: schema/parser.cpp [hunk 2]
```diff
@@ -739,15 +729,15 @@ SimpleType Parser::parseSimpleType(ParserContext *context, const QDomElement &el
                         ctItem.setName(
                                 st.name()
                                 + QLatin1String("ListItem")); // need to make something up, so that
-                                                              // the classname looks good
+                        // the classname looks good
```
Comment (dfaure-kdab) [inline schema/parser.cpp:732]: (same)

File: schema/parser.cpp [hunk 3]
```diff
@@ -739,15 +729,15 @@ SimpleType Parser::parseSimpleType(ParserContext *context, const QDomElement &el
...
                         st.setListTypeName(ctItem.qualifiedName());
                         d->mComplexTypes.append(ctItem);
                     } else if (typeName.localName() == QLatin1String("simpleType")) {
                         SimpleType stItem = parseSimpleType(context, typeElement);
                         stItem.setName(
                                 st.name()
                                 + QLatin1String("ListItem")); // need to make something up, so that
-                                                              // the classname looks good
+                        // the classname looks good
```
Comment (dfaure-kdab) [inline schema/parser.cpp:740]: (same)

**Review by dfaure-kdab — CHANGES_REQUESTED** [review]
Good idea.

File: schema/parser.cpp (hunk 1 above)
Comment (martonmiklos) [reply schema/parser.cpp]: Indeed, merging is not possible due to the Qt's clang-formatter 100 char line limit.

**Review by dfaure-kdab — APPROVED** [review]
I don't like line-length limits. We have very wide screens nowadays, it makes no sense anymore.

**Comment by martonmiklos** [comment]
> I don't like line-length limits. We have very wide screens nowadays, it makes no sense anymore.

Yeah I am also with the same opinion. Maybe we should tweak the .clang-formatter to exclude this limitation.

**Pull request merged**